playbooks/roles/csi-driver-install/tasks/main.yaml

```diff
@@ -0,0 +1,9 @@
+---
+# tasks file for csi driver config and install
+
+- name: Run csi driver configuration tasks
+  import_tasks: config.yaml
+
+- name: Run csi driver installation tasks
+  import_tasks: install.yaml
+  when: (resource_type != "") and (resource_version != "")
```

Comment:
Those variables will not be empty as you are defining the default values in this role.

Reply:
Also you are mentioning these 2 variables are mandatory.